content/docs/user-guide/managing-external-data.md

```diff
@@ -33,25 +31,32 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
+External outputs require an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+in the same external/remote file.
+
+> Note that [remote storage](/doc/command-reference/remote) is a separate
+> feature, and that external outputs are not pushed or pulled from/to DVC
+> remotes.
 
-> Avoid using the same [DVC remote](/doc/command-reference/remote) (used for
-> `dvc push`, `dvc pull`, etc.) for external outputs, because it may cause file
-> hash overlaps: the hash of an external output could collide with a hash
-> generated locally for another file with different content.
+> ⚠️ Avoid using the same DVC remote used for `dvc push`, `dvc pull`, etc. for
+> external outputs, because it may cause data collisions: the hash of an
+> external output could collide with that of a local file with different
+> content.
 
 ## Examples
 
-Let's take a look at:
+Let's take a look at the following operations on all the supported location
+types:
 
-1. Adding a `dvc remote` to use as cache for data in the external location, and
+1. Adding a `dvc remote` in the same location as the desired outputs, and
    configure it as external <abbr>cache</abbr> with `dvc config`.
-2. Tracking existing data on an external location with `dvc add` (this doesn't
-   download it). This produces a `.dvc` file with an external output.
-3. Creating a simple [stage](/doc/command-reference/run) that moves a local file
-   to the external location. This produces a stage with another external output
-   in `dvc.yaml`.
+2. Tracking existing data on the location using `dvc add` (`--external` option
```

Comment:
on the location - sounds strange a bit

Reply:
Yes. Going for "on the external location". Thanks